mail/src/main/java/com/sun/mail/util/logging/LogManagerProperties.java

```diff
@@ -17,6 +17,10 @@
 package com.sun.mail.util.logging;
 
 import java.io.*;
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodHandles.Lookup;
+import java.lang.invoke.MethodType;
```

Comment:
@jmehrens this is causing android sigtests to fail. Do you want me to rollback this PR from 1.6.x or do you prefer to fix it?

Reply:
@lukasj Looks like the Android API level 26 is where MethodHandles were added.  I'll update to use reflection for 1.x.  I should be able to get it patched tonight.